config/jobs/kubernetes/sig-arch/kubernetes-depstat.yaml

```diff
@@ -0,0 +1,27 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: kubernetes-depstat
+    decorate: true
+    always_run: true
+    skip_report: true
+    spec:
+      containers:
+      - image: golang
+        command: ["/bin/bash"]
+        args:
+        - -c
+        - |
+          set -euo; \
+          BASE_SHA="${PULL_BASE_SHA:-"09268c16853b233ebaedcd6a877eac23690b5190"}"; \
+          ARTIFACTS="${ARTIFACTS:-/tmp/artifacts}"; \
```

Comment:
@dims if I exclude these two lines and run `bash -c "<rest of the script>"` it works. With them I get this error:

```
mkdir: missing operand
Try 'mkdir --help' for more information.
```

This is because these variables don't get set when the entire script is passed to `bash -c`. 

What I can't seem to understand is why this is happening because individually running these two commands and then passing the rest of the script works.

P.S. I'll remove the hardcoded SHA before we merge.

Reply:
try this, note the quotes.
```
          BASE_SHA=${PULL_BASE_SHA:-"09268c16853b233ebaedcd6a877eac23690b5190"}; \
          ARTIFACTS=${ARTIFACTS:-/tmp/artifacts}; \
```

Reply:
Still not working 😞

Reply:
try `bash -c ''` (try single quote there with the change above too)

Reply:
This didn't work too :(

Trying something simpler like: `bash -c 'ARTIFACTS="/tmp/artifacts";'` is also not working.